apps/docs/src/app/core/component-docs/date-picker/examples/date-picker-allow-null-example.component.ts

```diff
@@ -4,7 +4,7 @@ import { DatePickerComponent, FdDate } from '@fundamental-ngx/core';
 @Component({
     selector: 'fd-date-picker-allow-null-example',
     template: `
-        <fd-date-picker [allowNull]="false" [type]="'single'" [(ngModel)]="date" [state]="isInvalid() ? 'invalid' : 'valid'"></fd-date-picker>
+        <fd-date-picker [allowNull]="false" [type]="'single'" [(ngModel)]="date" [state]="isInvalid() ? 'error' : 'success'"></fd-date-picker>
```

Comment:
It's not a big deal but some examples have an isInvalid method and others have isValid.  Could be confusing so I think try to avoid double negatives when possible